### PR TITLE
More tests for the frontend

### PR DESCRIPTION
**Pull request by tirr-c**

### CODE REVIEW

File: frontend/src/app/state/effects/user.spec.ts [hunk 1]
```diff
@@ -44,7 +52,8 @@ describe('UserEffects', () => {
 
   beforeEach(inject(
     [Actions, Http, Store, XHRBackend],
-    (actions$: Actions, http: Http, store: Store<State>, backend: MockBackend) => {
+    (actions$: Actions, http: Http, store: Store<State>, _backend: MockBackend) => {
+      backend = _backend;
```
Comment (skystar-p) [inline frontend/src/app/state/effects/user.spec.ts]: 이거 왜 이렇게 고친거야?

Comment (tirr-c) [reply frontend/src/app/state/effects/user.spec.ts]: 저 밑에서 `backend.connections`에다 subscribe를 더 해야 하는데 그러려면 `backend`를 가져가는 게 맞을 것 같아서? 걍 필요할 때마다 `inject`로 처리할까

Comment (skystar-p) [reply frontend/src/app/state/effects/user.spec.ts]: ㄴㄴ 이게 더 간단해보이는듯

**Comment by tirr-c** [comment]
없음!

**Comment by skystar-p** [comment]
@web-mighty-homu r+

**Comment by web-mighty-homu** [comment]
:pushpin: Commit 95b61d9 has been approved by `skystar-p`

**Comment by web-mighty-homu** [comment]
:hourglass: Testing commit 95b61d90d44cf563158cb1b7ff2ba676e4a872f2 with merge dd9150719738f688653fa5ec2048925c0674d7cc...

**Comment by web-mighty-homu** [comment]
:sunny: Test successful - [status-travis](https://travis-ci.org/swsnu/swpp17-team2/builds/301184122?utm_source=github_status&utm_medium=notification)
Approved by: skystar-p
Pushing dd9150719738f688653fa5ec2048925c0674d7cc to master...

**Pull request merged**